astropy/vo/samp/hub.py

```diff
@@ -188,9 +189,9 @@ def __init__(self, secret=None, addr=None, port=0, lockfile=None,
                 self._web_profile_server.register_introspection_functions()
                 log.info("Hub set to run with Web Profile support enabled.")
             except socket.error:
-                log.warn("Port {0} already in use. Impossible to run the "
-                         "Hub with Web Profile support.".format(self._web_port),
-                         SAMPWarning)
+                warnings.warn("Port {0} already in use. Impossible to run the "
+                              "Hub with Web Profile support.".format(self._web_port),
+                              SAMPWarning, AstropyWarning)
```

Comment:
This is already creating a `SAMPWarning` which is presumably a subclass of `AstropyWarning`.